ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/highLevelHumanoidControl/stateTransitions/FeetLoadedToWalkingStandTransition.java

```diff
@@ -19,12 +24,60 @@ public class FeetLoadedToWalkingStandTransition extends FeetLoadedTransition
+   public FeetLoadedToWalkingStandTransition(HighLevelControllerName nextStateEnum,
+                                             YoEnum<HighLevelControllerName> requestedState,
+                                             boolean waitForRequestToTransition,
+                                             SideDependentList<FootSwitchInterface> footSwitches,
+                                             double controlDT,
+                                             double totalMass,
+                                             double gravityZ,
+                                             HighLevelControllerParameters highLevelControllerParameters,
+                                             YoRegistry parentRegistry)
+   {
+      this(nextStateEnum,
+           requestedState,
+           waitForRequestToTransition,
+           (side) -> footSwitches.get(side).getMeasuredWrench(),
+           controlDT,
+           totalMass,
+           gravityZ,
+           highLevelControllerParameters,
+           parentRegistry);
+   }
+
+   public FeetLoadedToWalkingStandTransition(HighLevelControllerName nextStateEnum,
+                                             YoEnum<HighLevelControllerName> requestedState,
+                                             boolean waitForRequestToTransition,
+                                             ForceSensorDataHolderReadOnly forceSensorDataHolder,
+                                             SideDependentList<String> feetForceSensors,
+                                             double controlDT,
+                                             double totalMass,
+                                             double gravityZ,
+                                             HighLevelControllerParameters highLevelControllerParameters,
+                                             YoRegistry parentRegistry)
+   {
+      this(nextStateEnum,
+           requestedState,
+           waitForRequestToTransition,
+           (side) -> forceSensorDataHolder.getData(feetForceSensors.get(side)).getWrench(),
+           controlDT,
+           totalMass,
+           gravityZ,
+           highLevelControllerParameters,
+           parentRegistry);
+   }
+
+   public FeetLoadedToWalkingStandTransition(HighLevelControllerName nextStateEnum,
+                                             YoEnum<HighLevelControllerName> requestedState,
+                                             boolean waitForRequestToTransition,
+                                             Function<RobotSide, WrenchReadOnly> footSensors,
+                                             double controlDT,
+                                             double totalMass,
+                                             double gravityZ,
+                                             HighLevelControllerParameters highLevelControllerParameters,
                                              YoRegistry parentRegistry)
    {
```

Comment:
This adds a bunch of constructor options for using either the force data holder or the foot switches.